crates/runtime-configs/src/implementors/azapp.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::Result;
+use azure_app_configuration::{client::AzureAppConfigClient, search_label::SearchLabel};
+use futures::executor::block_on;
+
+use super::envvars::EnvVars;
+
+pub struct AzApp;
+
+fn make_client() -> Result<AzureAppConfigClient> {
+    Ok(AzureAppConfigClient::new(
+        String::from_utf8(EnvVars::get("AZAPPCONFIG_ENDPOINT")?)?,
+        String::from_utf8(EnvVars::get("AZAPPCONFIG_KEYID")?)?,
+        String::from_utf8(EnvVars::get("AZAPPCONFIG_KEYSECRET")?)?,
+    ))
+}
+
+impl AzApp {
+    pub fn get(key: &str) -> Result<Vec<u8>> {
+        let app_config_client = make_client()?;
+        let res: String = block_on(app_config_client.get_key_value(key, SearchLabel::All))
+            .expect("failed to get key")
+            .value;
+
+        Ok(res.as_bytes().to_vec())
+    }
+
+    pub fn set(key: &str, value: &[u8]) -> Result<()> {
+        let app_config_client = make_client()?;
+
+        block_on(app_config_client.set_key(
+            key,
+            std::str::from_utf8(value)?,
+            SearchLabel::For(""),
+            None,
+            None,
+        ))
+        .expect("failed to set key value");
+
+        Ok(())
+    }
+}
+
+#[cfg(test)]
+mod unittests {
+    use anyhow::Result;
+
+    use super::AzApp;
+
+    #[test]
+    fn set_then_get_test() -> Result<()> {
+        AzApp::set("key", "value".as_bytes())?;
```

Comment:
I don't think you can run `AzApp::set` as unittest because it require connection to external provider. Unit test is supposed to be self-contained. How about port this to our integration test where you have access to environment variables?

Reply:
I've moved this to integration tests ✅